wicket/src/ui/panes/update.rs

```diff
@@ -829,8 +841,150 @@ impl UpdatePane {
                 ]);
                 let paragraph = Paragraph::new(text)
                     .alignment(Alignment::Center)
-                    .block(block.clone().title("UPDATE READY"));
+                    .block(block.clone().title("UPDATE READY").borders(
+                        Borders::LEFT | Borders::RIGHT | Borders::TOP,
+                    ));
                 frame.render_widget(paragraph, self.status_view_main_rect);
+                frame.render_widget(
+                    help_text(&self.not_started_help).block(block.clone()),
+                    self.help_rect,
+                );
+                frame.render_widget(
+                    BoxConnector::new(BoxConnectorKind::Bottom),
+                    self.status_view_main_rect,
+                );
+            }
+            UpdateItemState::StartUpdate { response } => {
```

Comment:
I think it may be a bit easier to read if you factor out the rect sizing and text construction to named functions.

Reply:
Yeah, I did a big refactor and moved everything to a `StatusView` widget wrapper.